### PR TITLE
Scripts for generating dataset for precise from existing dataset

### PR DESCRIPTION
**Pull request by ljj7975**

### CODE REVIEW

File: howl/run/generate_precise_roc.py [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+            print("Not a float: ", name)
+
+    clean_dev_far, clean_dev_frr, clean_test_far, clean_test_frr = get_metrics(args.exp_type, clean_wb, thresholds)
+    noisy_dev_far, noisy_dev_frr, noisy_test_far, noisy_test_frr  = get_metrics(args.exp_type, noisy_wb, thresholds)
+
+    precise_clean_dev_far, precise_clean_dev_frr, precise_clean_test_far, precise_clean_test_frr = get_metrics(args.exp_type, precise_clean_wb, thresholds)
+    precise_noisy_dev_far, precise_noisy_dev_frr, precise_noisy_test_far, precise_noisy_test_frr  = get_metrics(args.exp_type, precise_noisy_wb, thresholds)
+
+
+    # print("thresholds:", thresholds)
```
Comment (edwinzhng) [inline howl/run/generate_precise_roc.py:102]: Are these commented print statements needed?

File: howl/run/generate_precise_roc.py [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+
+def get_metrics(exp_type, wb, thresholds):
+    dev_far = []
+    dev_frr = []
+
+    test_far = []
+    test_frr = []
+    if exp_type == "hey_ff":
+        total_train_len = 111928.04543750011
```
Comment (edwinzhng) [inline howl/run/generate_precise_roc.py:20]: Just curious, where do these numbers come from?

**Review by edwinzhng — APPROVED** [review]
LGTM!

File: README.md [hunk 3]
```diff
@@ -147,4 +153,5 @@ python -m howl.run.eval_wake_word_detection --num_models n --hop_size < number b
 We also provide a script for generating ROC curve. `exp_timestamp` can be found from the reports generated from previous command
 ```bash
 python -m howl.run.generate_roc --exp_timestamp < experiment timestamp > --exp_type < hey_firefox | hey_snips >
+>>>>>>> 95bb4c98b1e3c4d67016f9f8d1ad30dfa6d42fd1
```
Comment (Mic92) [inline README.md:156]: This should be removed.

**Pull request merged**